### PR TITLE
V2 Update attach bindings to use Readers/Writers vs chan

### PR DESCRIPTION
**Pull request by jwhonce**
Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: pkg/bindings/containers/containers.go [hunk 1]
```diff
@@ -373,11 +378,18 @@ func Attach(ctx context.Context, nameOrId string, detachKeys *string, logs, stre
 	}
 	defer response.Body.Close()
 
-	ctype := response.Header.Get("Content-Type")
-	upgrade := response.Header.Get("Connection")
+	// if stdin != nil {
+	// 	// Copy to stdin
+	// }
```
Comment (vrothberg) [inline pkg/bindings/containers/containers.go:383]: Looks like we don't need this

**Comment by baude** [comment]
lgtm

**Pull request merged**